src/main/java/seedu/address/model/tag/TutorialTag.java

```diff
@@ -0,0 +1,16 @@
+package seedu.address.model.tag;
+
+/**
+ * Represents a tutorial tag.
+ */
+public class TutorialTag extends Tag {
```

Comment:
What is the difference between TutorialTag and AttendanceTag? I thought we were marking attendance for tutorials?

Reply:
Attendance is meant for individual sessions of one tutorial group, such as W1Tutorial, W2Tutorial. On the other hand TutorialTag represents the tutorial session a student is assigned to/a tutor is available during

Reply:
Thanks for the clarification!